config-editor/config-editor-ui/src/app/services/config-loader.service.ts

```diff
@@ -412,6 +415,14 @@ export class ConfigLoaderService {
       .pipe(map(result => result));
   }
 
+  setConfigTester(testConfig: TestConfigSpec) {
```

Comment:
why is this stored in this service? Backend support tetsing based on name of teh tester. Thsi functionality needs to be reworked